src/core/prometheus.cc

```diff
@@ -875,7 +1044,7 @@ bool is_accept_protobuf(const std::string& accept) {
 class metrics_handler : public httpd::handler_base  {
     sstring _prefix;
     config _ctx;
-    static std::function<bool(const mi::labels_type&)> _true_function;
+    static const std::function<bool(const mi::labels_type&)> _true_function;
```

Comment:
Is this `const` addition an essential part of this patch, or just happened to be here?

Reply:
It's not essential, I just noticed this should be const when I was evaluating how much the function eval costs (not enough to bother, so only the const was changed). Happy to remove it. I don't think it makes  a perf difference.

Reply:
Please

Reply:
Removed in latest push.